src/aliases.js

```diff
@@ -60,7 +61,7 @@ const ALIASES = {
 
     'edge': {
         nameRe:             /edge/i,
-        cmd:                '--new-window',
+        cmd:                '--new-window --disable-background-timer-throttling',
```

Comment:
@alexey-lin Do we really need this for Edge?

Reply:
Yes, we do. I tested Edge with and without this option under macOS. The option works and the difference is remarkable.